Sage/carbon
# fix(action-popover): resolve issue with Select component closing parent Dialog on escape key press - FE-5688

### Proposed behaviour

This PR fixes an issue where closing an ActionPopover nested in Dialog with an escape keypress closes both the ActionPopover and the Dialog

### Checklist




#### QA


### Additional context



### Testing instructions

When ActionPopover menu is open, pressing an escape key should only close the ActionPopover menu, Dialog should stay open.

https://codesandbox.io/s/carbon-quickstart-forked-ycydfb
